packages/jobs/lib/activities.ts

```diff
@@ -466,14 +479,21 @@ export async function runPostConnectionScript(args: PostConnectionScriptArgs): P
 
     const context: Context = Context.current();
 
-    const syncRun = new syncRunService({
+    const syncRun = new SyncRunService({
         bigQueryClient,
         integrationService,
         recordsService,
         slackService,
         writeToDb: true,
         nangoConnection,
-        syncName: name,
+        syncConfig: {
+            sync_name: name,
+            file_location,
+            models: [],
+            track_deletes: false,
+            type: SyncConfigType.SYNC,
+            version: '0' // TODO: pass the correct version
+        },
```

Comment:
Post Script Connection is the only weird kid regarding run service. My take with the future refacto, we should have one base class and N class per script type so they only require what's relevant for them